Comp2025/src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -104,63 +104,40 @@ public void initialize( )
   {
     DataLogManager.log(String.format("%s: Subsystem initialized!", getSubsystem( )));
 
-    LimelightHelpers.setLEDMode_ForceOff(kLLName);
-    LimelightHelpers.setStreamMode_PiPSecondary(kLLName);
+    LimelightHelpers.setLEDMode_ForceOff(Constants.kLLLeftName);
+    LimelightHelpers.setLEDMode_ForceOff(Constants.kLLRightName);
+    LimelightHelpers.setStreamMode_PiPSecondary(Constants.kLLLeftName);
+    LimelightHelpers.setStreamMode_PiPSecondary(Constants.kLLRightName);
 
-    SetIMUMode(imuMode.EXTERNAL_SEED);
+    SetIMUMode(imuMode.EXTERNAL_SEED, "limelight-left");
```

Comment:
We probably want to set the IMU mode for BOTH limelights, not just the left one.